spec/lib/court_data_adaptor/resource/defendant_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   it { is_expected.to respond_to(:prosecution_case_reference, :prosecution_case_reference=) }
-  it { is_expected.to respond_to(:name) }
+  it { is_expected.to respond_to(:id, :name, :date_of_birth, :maat_reference) }
```

Comment:
Not a blocker: if we are adding in these respond_to matchers for lib/court_data_adaptor/resource/defendant.rb should we also add those associated with linking/unlinking (i.e. user_name,  unlink_reason_code, unlink_reason_text)

Reply:
makes sense, especially since those properties are needed for the update/linking of a defendant

Reply:
done